api/patient.py

```diff
@@ -8,39 +8,55 @@
 @apiParam {String}   last_name			Mandatory last name.
 @apiParam {String}   first_name_pinyin	Mandatory first name pinyin.
 @apiParam {String}   last_name_pinyin	Mandatory last name pinyin.
-@apiParam {String}   gender="M"			Mandatory gender with default "M".
+@apiParam {String}   gender				Mandatory gender with default "M".
 @apiParam {DateTime} birthdate			Mandatory birthdate.
 @apiParam {String}   relationship		Mandatory replationship with user.
 @apiParam {String}	 passport_number 	Mandatory passport number.
-@apiParam {String}	 note				Optional note.
+@apiParam {String}	 [note]				Optional note.
 
-@apiSuccess {String} msg		Successfully create patient. 
+@apiSuccess {String} patient_id			On Successful creation, return patient_id. 
 
 @apiVersion 0.0.0
 """
 
 """
-@api {post} /patient/:user_id/update Update patient
+@api {post} /patient/:patient_id/update Update patient
 @apiName UpdatePatient
 @apiGroup Patient
 
-@apiParam {Number}   user_id			Mandatory unique User ID.
-@apiParam {Number}	 patient_id			Mandatory unique Patient ID.
-@apiParam {String}   first_name 		Mandatory first name.
-@apiParam {String}   last_name			Mandatory last name.
-@apiParam {String}   first_name_pinyin	Mandatory first name pinyin.
-@apiParam {String}   last_name_pinyin	Mandatory last name pinyin.
-@apiParam {String}   gender="M"			Mandatory gender with default "M".
-@apiParam {DateTime} birthdate			Mandatory birthdate.
-@apiParam {String}   relationship		Mandatory replationship with user.
-@apiParam {String}	 passport_number 	Mandatory passport number.
-@apiParam {String}	 note				Optional note.
+@apiParam {Number}	 patient_id				Mandatory unique Patient ID.
+@apiParam {String}   [first_name]	 		Mandatory first name.
```

Comment:
These fields could be optional in update endpoint. -- Although the frontend will always include all of them in the update post request.